lib/locator.js

```diff
@@ -116,7 +116,7 @@ class Locator {
   }
 
   withChild(locator) {
-    const xpath = sprintf('%s[//%s]', this.toXPath(), removePrefix((new Locator(locator, 'css')).toXPath()));
+    const xpath = sprintf('%s//descendant::%s', this.toXPath(), removePrefix((new Locator(locator, 'css')).toXPath()));
```

Comment:
How about change to `'%s//child::%s'` and create separate method `withDescendant` ?
Looks more "xpathier", than "child" as any descendant?